packages/runtime/src/templates/edge/runtime.ts

```diff
@@ -63,10 +63,12 @@ const handler = async (req: Request, context: Context) => {
     return
   }
 
-  const geo = {
+  const geo: RequestData['geo'] = {
     country: context.geo.country?.code,
     region: context.geo.subdivision?.code,
     city: context.geo.city,
+    latitude: context.geo.latitude?.toString(),
```

Comment:
Converting to string her as this is what Next.js is expecting.

Reply:
How about adding this to `packages/next/src/middleware/request.ts`:

```typescript
export type GeoData = NextRequest['geo'] & {
  timezone?: string
}
```

... and then type the getter `get geo(): GeoData`

Reply:
Maybe we should declare our own `NextRequest` type inside `@netlify/next` that extends the original one, and export it from there. We can also type the `geo()` getter in `MiddlewareRequest`